tests/unittests/utils/test_muse_glimmer.py

```
@@ -0,0 +1,317 @@
+
+        assert len(detections) == 0
+
+    def test_parse_non_finite_coordinates(self):
+        processor = MuseGlimmerOutputProcessor()
+
+        big_int = "9" * 400
+        for raw in (
+            '[{"label": "cat", "bbox_2d": [NaN, 0, 500, 500]}]',
+            '[{"label": "cat", "bbox_2d": [0, Infinity, 500, 500]}]',
+            '[{"label": "cat", "bbox_2d": [0, -Infinity, 500, 500]}]',
+            '[{"label": "cat", "bbox_2d": [%s, 0, 500, 500]}]' % big_int,
```

Ruff (0.16.1)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)